rules.ini

```
@@ -248,7 +248,7 @@ CEF = (?:^|/)libcef\.(?:dll|so)$
 Coherent_Gameface_OR_Prysm = (?:^|/)cohtml\.windowsdesktop\.dll$
 CRIWARE[] = (?:^|/)data(?:1|000)?\.cpk$
 CRIWARE[] = \.(?:sfd|usm|adx|acb|awb)$
-cURL = curl(?:module|lib|d|-4|-ttv|-x64|64|_pluginw64_release)?\.(?:dll|exe|lib|so|so\.4|so\.4\.5\.0)$
+cURL = curl(?:module|lib|d|-?[34]|-ttv|-x64|64|_pluginw64_release)?\.(?:dll|exe|lib|so|so\.[34]|so\.[34]\.[0-9]+\.[0-9]+)$
```

```suggestion
cURL = curl(?:module|lib|d|-?[34]|-ttv|-x64|64|_pluginw64_release)?\.(?:dll|exe|lib|so|so\.[34]|so\.[34]\.[0-9]{1,3}\.[0-9]{1,3})$
``` 
we prefer not using `+` in regex
and i'm not even sure if looking for 3 numbers is required, 2 is probably enough

---

3 seems pretty innocuous. Done.